neuraxle/metaopt/auto_ml.py

```diff
@@ -644,39 +581,20 @@ def _load_best_model(self) -> BaseStep:
 
```

Comment:
This seems broken, resolve to issue
```
        trials: Trials = self.hyperparams_repository.load_all_trials(TRIAL_STATUS.SUCCESS)	        trials: Trials = self.hyperparams_repository.load_all_trials(TRIAL_STATUS.SUCCESS)
        auto_ml_algorithm: AutoMLAlgorithm = self.wrapped	        auto_ml_algorithm: AutoMLAlgorithm = self.wrapped
        best_hyperparams = auto_ml_algorithm.get_best_hyperparams(trials)	        best_hyperparams = auto_ml_algorithm.get_best_hyperparams(trials)
        auto_ml_algorithm = auto_ml_algorithm.update_hyperparams(best_hyperparams)	        auto_ml_algorithm = auto_ml_algorithm.update_hyperparams(best_hyperparams)
```

Reply:
https://github.com/Neuraxio/Neuraxle/issues/251